Test/cloudmake/CloudMake-ParallelBuilds.dfy

```diff
@@ -150,10 +150,10 @@ abstract module M0 {
 
   datatype Reason = rCompatibility | rValidity
 
-  type Path(==,0)  // TODO: this is a good candidate for a "ghost (0)" marking (meaning nonempty type)
+  type Path(==,00)
   function Loc(cmd: string, deps: set<Path>, exp: string): Path
 
-  type Artifact(0)  // TODO: this is a good candidate for a "ghost (0)" marking (meaning nonempty type)
+  type Artifact(0)
```

Comment:
Given the comment, I would have expected 00 here

Reply:
Done. (Due to an intermediate bug, I temporarily changed it back to `(0)` and then forgot to re-change to it `(00)`.)